Settings/PricePartSettings.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace OrchardCore.Commerce.Settings
+{
+    public enum CurrencySelectionModeEnum
```

Comment:
One last thing, can you move the enum to a separate file?

Reply:
Sure.